Sources/Views/Cells/MessageContentCell.swift

```diff
@@ -60,6 +60,9 @@ open class MessageContentCell: MessageCollectionViewCell {
         return label
     }()
 
+    // Should only add customized subviews - don't change accessoryView itself.
+    open var accessoryView: UIView = .init()
```

Comment:
Can you change this to `open var accessoryView: UView = UIView()`?

Reply:
sure